flexible/errorreporting/src/main/java/com/example/flexible/errorreporting/ErrorReportingExample.java

```diff
@@ -53,13 +53,12 @@ private void logCustomErrorEvent() {
     try (ReportErrorsServiceClient reportErrorsServiceClient = ReportErrorsServiceClient.create()) {
       // custom exception logged using the API
       Exception e = new Exception("custom event reported using the API");
-      // Events reported using the API must contain a stack trace message
-      StringWriter stackTrace = new StringWriter();
-      e.printStackTrace(new PrintWriter(stackTrace));
+      StringWriter sw = new StringWriter();
+      e.printStackTrace(new PrintWriter(sw));
       ReportedErrorEvent errorEvent =
```

Comment:
Just weird. 56-57 & L61

Reply:
If we are going to do this, we really should explain what's happening.

e.printStackTrace(PrintWriter(StringWriter))
.setMessage(StringWriter.toString())